ballet/eng/misc.py

```diff
@@ -4,6 +4,8 @@
 
 from ballet.eng.base import BaseTransformer, SimpleFunctionTransformer
 from ballet.util import get_arr_desc
+from scipy.special import boxcox1p
```

Comment:
sort imports according to isort rules: https://github.com/timothycrosley/isort#how-does-isort-work

just use the `make fix-lint` target